test/utils.c

```diff
@@ -130,6 +130,20 @@ TEST test_string_to_path(void)
         PASS();
 }
 
+TEST test_string_to_time(void)
+{
+        char *input[] = { "5000 ms", "5000ms",  "100", "10s",   "2m",    "11h",      "9d", NULL };
+        gint64  exp[] = {      5000,     5000, 100000, 10000, 120000, 39600000, 777600000 };
+
+        int i = 0;
+        while (input[i]){
+                ASSERT_EQ(string_to_time(input[i]), exp[i]*1000);
```

Comment:
From the recent failed test, the error for this assertion is unreadable:
```
FAIL test_string_to_time: string_to_time(input[i]) != exp[i]*1000 (test/utils.c:140)
```

[`ASSERT_EQ_FMT`](https://github.com/silentbicycle/greatest#assert_eq_fmtexpected-actual-format) should be used to explain what was tested and what failed.

Reply:
I'm already in it ;-)

Reply:
But thanks for the hint with `ASSERT_EQ_FMT`. That's gold